modules/lm-coursier/src/main/scala/lmcoursier/definitions/ToCoursier.scala

```diff
@@ -36,6 +35,27 @@ object ToCoursier {
       module.attributes
     )
 
+  def moduleMatcher(matcher: ModuleMatcher): coursier.util.ModuleMatcher =
+    if (matcher == ModuleMatcher.all) coursier.util.ModuleMatcher.all
```

Comment:
Do we really need that `if`? The `else` case should work in both cases.

Reply:
I thought I'd avoid extra allocation, but it seems like `all` is using `def` so I guess it doesn't matter.
I can remove the if part.

Reply:
6355338735046930e5af448ef3f3a976e98c548d